Realm.PCL/RealmObjectPCL.cs

```diff
@@ -36,7 +36,7 @@ public class RealmObject
         private Realm _realm;  // may not be used but wanted it included in definition of IsManaged below.
 
         /// <summary>
-        /// Allows you to check if the object has been associated with a Realm, either at creation or via Realm.Manage.
+        /// Allows you to check if the object has been associated with a Realm, either at creation or via Realm.Add.
         /// </summary>
         public bool IsManaged => _realm != null;
```

Comment:
In Java it's [`isManaged`](https://github.com/realm/realm-java/blob/master/realm/realm-library/src/main/java/io/realm/RealmObject.java#L263), while the `Add` operation is called [`copyToRealm`](https://github.com/realm/realm-java/blob/master/realm/realm-library/src/main/java/io/realm/Realm.java#L882) so a little inconsistency seems fine (or at least it won't be us to set a precedent). Personally, I think that it's fine to stick with `IsManaged` and the `XXXManagedXXXException`s.